supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/statuspolling/JesApiQueryManager.scala

```diff
@@ -53,13 +53,13 @@ class JesApiQueryManager(val qps: Int Refined Positive) extends Actor with Actor
   }
 
   private def handleQueryFailure(failure: JesApiQueryFailed) = failure match {
-    case JesApiQueryFailed(request, _) if request.failedAttempts < maxRetries =>
-      val nextRequest = request.withFailedAttempt
+    case failed: JesApiQueryFailed if failed.query.failedAttempts < maxRetries =>
+      val nextRequest = failed.query.withFailedAttempt
       val delay = nextRequest.backoff.backoffMillis.millis
       context.system.scheduler.scheduleOnce(delay, self, nextRequest)
       ()
-    case JesApiQueryFailed(request, _) =>
-      request.requester ! failure
+    case failure: JesApiQueryFailed =>
+      failure.query.requester ! failure
```

Comment:
I know I wrote it originally, but this whole `match` should probably just be an `if` statement. e.g.
```
handleQueryFailure(failure: JesApiQueryFailed) = if (failure.query.failedAttempts > maxRetries) {
...
} else {
...
}
```